blockchain/blockchain-wallet-v4-frontend
# Handle SFOX profile failure

## Description
696

## Change Type
Please enter one or more of the following: 
- Bug Fix

## Testing Steps
Fail the address/id verification step and close the tray. Failure view should not render.

## Code Checklist
